packages/devtools/src/composite.ts

```diff
@@ -63,10 +70,68 @@ function assertModelsHaveCommits(
   }
 }
 
-function assertSupportedModelController(model: Model): void {
-  if (!model.metadata.controller.startsWith('did:key:')) {
+function assertSupportedWriteModelController(model: Model, ceramic: CeramicApi): void {
+  const unsupported = `Unsupported model controller ${model.metadata.controller}`
+  if (model.metadata.controller.startsWith('did:pkh:')){
+    if (ceramic.context.did == null) {
+      throw new Error(
+        `${unsupported}, did missing from ceramic context`
+      )
+    }
+    if (!ceramic.context.did.hasCapability()) {
+      throw new Error(
+        `${unsupported}, only did:pkh with CACAO is supported`
+      )
+    }
+    const cacao: Cacao = ceramic.context.did.capability(); 
+    if (cacao.p.exp == null) {
```

Comment:
I think this logic is backwards? We should only allow creation if ocap doesn't have expiry?
```suggestion
    if (cacao.p.exp !== null) {
```